substrate-relay/src/bridges/rococo_westend/asset_hub_rococo_messages_to_bridge_hub_westend.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! AssetHubRococo-to-BridgeHubWestend messages sync entrypoint.
+
+use relay_asset_hub_rococo_client::AssetHubRococo;
+use relay_bridge_hub_westend_client::BridgeHubWestend;
+use substrate_relay_helper::{
+	cli::bridge::{CliBridgeBase, MessagesCliBridge},
+	messages::SubstrateMessageLane,
+	UtilityPalletBatchCallBuilder,
+};
+
+pub struct AssetHubRococoToBridgeHubWestendMessagesCliBridge {}
+
+impl CliBridgeBase for AssetHubRococoToBridgeHubWestendMessagesCliBridge {
+	type Source = AssetHubRococo;
+	type Target = BridgeHubWestend;
+}
```

Comment:
everywhere here we need to have AssetHubRococo to AssetHubWestend (no BridgeHub here),
and also the file should be:
```
substrate-relay/src/bridges/rococo_westend/asset_hub_rococo_messages_to_bridge_hub_westend.rs
substrate-relay/src/bridges/rococo_westend/asset_hub_rococo_messages_to_asset_hub_westend.rs
```

```suggestion
pub struct AssetHubRococoToAssetHubWestendMessagesCliBridge {}

impl CliBridgeBase for AssetHubRococoToAssetHubWestendMessagesCliBridge {
	type Source = AssetHubRococo;
	type Target = AssetHubWestend;
}
```